diff -urN Terminal-0.2.8/configure.in Terminal-0.2.8.new/configure.in
--- Terminal-0.2.8/configure.in	2007-12-02 15:16:13.000000000 +0100
+++ Terminal-0.2.8.new/configure.in	2007-12-07 15:28:44.000000000 +0100
@@ -71,7 +71,7 @@
 dnl ******************************
 dnl *** Check for i18n support ***
 dnl ******************************
-XDT_I18N([ar be ca cs de dz el en_GB eo es eu fi fr gl he hu it ja ko lt lv nb_NO nl pa pl pt_BR pt_PT ro ru sk sq sv tr uk ur vi zh_CN zh_TW])
+XDT_I18N([ar be ca cs de dz el en_GB eo es eu fi fr gl he hu it ja ko lt lv nb nl pa pl pt_BR pt ro ru sk sq sv tr uk ur vi zh_CN zh_TW])
 
 dnl ********************************
 dnl *** Check for X11 (optional) ***
diff -urN Terminal-0.2.8/po/LINGUAS Terminal-0.2.8.new/po/LINGUAS
--- Terminal-0.2.8/po/LINGUAS	2007-12-02 15:15:46.000000000 +0100
+++ Terminal-0.2.8.new/po/LINGUAS	2007-12-07 15:28:30.000000000 +0100
@@ -1,2 +1,2 @@
 # set of available languages (in alphabetic order)
-ar be ca cs de dz el en_GB eo es eu fi fr gl he hu it ja ko lt lv nb_NO nl pa pl pt_BR pt_PT ro ru sk sq sv tr uk ur vi zh_CN zh_TW
+ar be ca cs de dz el en_GB eo es eu fi fr gl he hu it ja ko lt lv nb nl pa pl pt_BR pt ro ru sk sq sv tr uk ur vi zh_CN zh_TW
