--- ./po/LINGUAS.org	2008-10-27 23:43:25.479961121 +0100
+++ ./po/LINGUAS	2008-10-27 23:47:22.892645757 +0100
@@ -1,2 +1,2 @@
 # set of available languages (in alphabetic order)
-ar be ca cs da de dz el en_GB eo es et eu fi fr gl he hu id it ja ko ku lt lv nb_NO nl pa pl pt_BR pt_PT ro ru sk sq sv tr uk ur vi zh_CN zh_TW
+ar be ca cs da de dz el en_GB eo es et eu fi fr gl he hu id it ja ko ku lt lv nb nl pa pl pt_BR pt ro ru sk sq sv tr uk ur vi zh_CN zh_TW
--- ./configure.in.org	2008-10-27 23:43:25.479961121 +0100
+++ ./configure.in	2008-10-27 23:47:01.999939361 +0100
@@ -71,7 +71,7 @@ AC_CHECK_HEADERS([ctype.h errno.h limits
 dnl ******************************
 dnl *** Check for i18n support ***
 dnl ******************************
-XDT_I18N([ar be ca cs da de dz el en_GB eo es et eu fi fr gl he hu id it ja ko ku lt lv nb_NO nl pa pl pt_BR pt_PT ro ru sk sq sv tr uk ur vi zh_CN zh_TW])
+XDT_I18N([ar be ca cs da de dz el en_GB eo es et eu fi fr gl he hu id it ja ko ku lt lv nb nl pa pl pt_BR pt ro ru sk sq sv tr uk ur vi zh_CN zh_TW])
 
 dnl ********************************
 dnl *** Check for X11 (optional) ***
